reference/docs/conversion/sakai_12_mysql_conversion.sql

```diff
@@ -246,3 +245,123 @@ ALTER TABLE SAKAI_POSTEM_STUDENT MODIFY COLUMN username varchar(99), DROP INDEX
 
 -- #3431 Add final grade mode setting
 ALTER TABLE GB_GRADEBOOK_T ADD COLUMN FINAL_GRADE_MODE bit(1) NOT NULL DEFAULT b'0';
+
+-- BEGIN 3432 Grade Points Grading Scale
+-- add the new grading scale
+INSERT INTO gb_grading_scale_t (id, object_type_id, version, scale_uid, name, unavailable)
+VALUES (gb_grading_scale_s.nextval, 0, 0, 'GradePointsMapping', 'Grade Points', 0);
```

Comment:
Hm I hit `Unknown column 'gb_grading_scale_s.nextval' in 'field list'` when I run the first INSERT.  Perhaps I'm missing something?